File: src/main/java/team4/footwithme/chat/api/request/ChatroomRequest.java
package team4.footwithme.chat.api.request;

import jakarta.validation.constraints.NotNull;
import team4.footwithme.chat.service.request.ChatroomServiceRequest;

public record ChatroomRequest(
        @NotNull(message = "채팅방 이름은 필수입니다.")
        String name
) {
    public ChatroomServiceRequest toServiceRequest() {
        return new ChatroomServiceRequest(name);
    }
}

File: src/main/java/team4/footwithme/chat/api/request/ChatroomRequest.kt
package team4.footwithme.chat.api.request;

import jakarta.validation.constraints.NotBlank;
import team4.footwithme.chat.service.request.ChatroomServiceRequest;

public record ChatroomRequest(
    @NotBlank(message = "채팅방 이름은 필수입니다.")
    String name
) {
    public ChatroomServiceRequest toServiceRequest() {
        return new ChatroomServiceRequest(name);
    }
}
